scene/gui/menu_button.cpp

```diff
@@ -146,16 +148,29 @@ void MenuButton::_notification(int p_what) {
 	}
 }
 
+bool MenuButton::_set(const StringName &p_name, const Variant &p_value) {
+	return popup->_set(p_name, p_value);
+}
+
+bool MenuButton::_get(const StringName &p_name, Variant &r_ret) const {
+	return popup->_get(p_name, r_ret);
+}
+
+void MenuButton::_get_property_list(List<PropertyInfo> *p_list) const {
+	popup->_get_property_list(p_list);
```

Comment:
I don't think is is a good idea to get the properties list directly like that. If we end up with a properties with colliding names, it might create very unexpected bug.
You should retrieve the list, then prefix each property with something like "popup/" to be sure no property in PopupMenu will override a property in MenuButton.

Reply:
Done. Much better this way